installation/installing-images/README.md

```diff
@@ -14,6 +14,8 @@ Raspberry Pi have developed a graphical SD card writing tool that works on Mac O
 - Choose  the SD card you wish to write your image to.
 - Review your selections and click 'WRITE' to begin writing data to the SD card.
 
+**Note**: if using the Imager on Windows 10 with Controlled Folder Access enabled, you will need to explicitly allow the Imager permission to write the SD card. If this is not done, the Imager will fail with a "failed to write" error.
```

Comment:
IMHO "the Imager" should be replaced with "Raspberry Pi Imager"

Reply:
Also, I've never used Windows 10 myself, but do the steps needed to "explicitly allow Raspberry Pi Imager permission to write the SD card" need to be specifically explained, or is the process obvious?

Reply:
Good question. IMO if someone has configured Windows CFA to be enabled, they probably know how to disable it for this particular executable. These days there's a pop-up that appears when the permission violation takes place that you can then click on to change the settings for that process, hence it's not _too_ opaque.